go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/open-feature/go-sdk
 
-go 1.21
```

Comment:
I'm reluctant to switch to Go `1.23` since  `1.22` is still supported.

FYI @toddbaert @thomaspoignant @lukas-reining

Reply:
It's not supported actually. Go 1.24 and 1.23 are the currently supported versions of Go. Go 1.22 stopped being supported as soon as 1.24.0 was released. This is standard.
https://go.dev/doc/devel/release

Reply:
I misread the policy, sorry. I thought it was the current release plus the two prior. For what's worth, the [OTel SDK only supports Go 1.23 and 1.24](https://github.com/open-telemetry/opentelemetry-go#compatibility).

I'll defer to the @open-feature/technical-steering-committee on this since changing major runtime versions can have a significant impact on consumers.

Reply:
No worries. Issue #349 has some additional details.